.github/workflows/build_recipes.yaml

```diff
@@ -38,19 +81,19 @@ jobs:
 
       - name: apt-install playwright dependencies
         run: |
-            sudo apt-get update -y && sudo apt-get install -y  \
-            libwoff1  libvpx9  libevent-2.1-7t64 libopus0 libgstreamer-plugins-base1.0-0 \
-            libgstreamer-gl1.0-0 \
-            libgstreamer-plugins-bad1.0-0 \
-            libflite1 \
-            libavif16 \
-            libharfbuzz-icu0 \
-            libsecret-1-0 \
-            libhyphen0 \
-            libmanette-0.2-0 \
-            libgles2 \
-            gstreamer1.0-libav \
-            libgtk-4-1
+          sudo apt-get update -y && sudo apt-get install -y  \
+          libwoff1  libvpx9  libevent-2.1-7t64 libopus0 libgstreamer-plugins-base1.0-0 \
+          libgstreamer-gl1.0-0 \
+          libgstreamer-plugins-bad1.0-0 \
+          libflite1 \
+          libavif16 \
+          libharfbuzz-icu0 \
+          libsecret-1-0 \
+          libhyphen0 \
+          libmanette-0.2-0 \
+          libgles2 \
+          gstreamer1.0-libav \
+          libgtk-4-1
```

Comment:
Please bear with me a bit - My editor is making these changes automatically. I am still figuring out